MapboxNavigation/PollyVoiceController.swift

```diff
@@ -96,7 +95,11 @@ public class PollyVoiceController: RouteVoiceController {
             input.voiceId = voiceId
         }
         
-        input.text = "<speak><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>\(text)</prosody></speak>"
+        let wrappedSSML = text
+            .replacingOccurrences(of: "<speak>", with: "<speak><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>")
+            .replacingOccurrences(of: "</speak>", with: "</prosody></speak>")
```

Comment:
Is it possible to nest `<prosody>` tags in SSML? I could imagine a future in which the Directions API decides to pronounce some portion of an instruction slightly louder than another portion.